xml/System.Security.Cryptography.Pkcs/SignedCms.xml

```diff
@@ -744,7 +744,7 @@ The following permissions are required to access the signature key:
           <paramref name="signer" /> is <see langword="null" />.</exception>
         <exception cref="T:System.Security.Cryptography.CryptographicException">A cryptographic operation could not be completed.</exception>
         <exception cref="T:System.InvalidOperationException">.NET Framework only: A signing certificate is not specified.</exception>
-        <exception cref="T:System.PlatformNotSupportedException">.NET Core only: A signing certificate is not specified.</exception>
+        <exception cref="T:System.PlatformNotSupportedException">.NET Core only: This member is not supported.</exception>
```

Comment:
@bartonjs can you please review this change?
It seems this exception is thrown in a specific condition for .NET Core so I don't think the change is correct but I'd like to double check with you

Reply:
I agree, it was already correct and this change is incorrect.